src/timeline/Timeline.js

```diff
@@ -87,6 +103,38 @@ export default class Timeline extends React.PureComponent {
     }, 1);
   }
 
+  currentTimeOffset() {
+    const offset = 100;
+    const {start = 0} = this.props;
+    const timeNowHour = moment().hour();
+    const timeNowMin = moment().minutes();
+
+    return offset * (timeNowHour - start) + (offset * timeNowMin) / 60;
+  }
+
+  _renderCurrentTimeIndicator() {
+    // currentDateForTimeIndicator format YYYY-MM-DD, e.g. 2020-11-06
+    if (typeof this.props.currentDateForTimeIndicator === 'undefined') {
+      return;
+    }
+
+    if (this.props.currentDateForTimeIndicator === moment().format('YYYY-MM-DD')) {
```

Comment:
`currentDateForTimeIndicator` is a long variable name, you may consider renaming this